src/en/weebcentral/src/eu/kanade/tachiyomi/extension/en/weebcentral/WeebCentral.kt

```diff
@@ -163,7 +167,7 @@ class WeebCentral : ParsedHttpSource() {
         element.selectFirst("svg")?.attr("stroke")?.also { stroke ->
             scanlator = when (stroke) {
                 "#d8b4fe" -> "Official"
-                "#4C4D54" -> "Unofficial"
+                "#4C4D54" -> "unknown"
```

Comment:
just remove this, so only "official" appears when necessary

Reply:
I'd prefer to have it in, as in the current state, the scanlator filter works on a blacklist basis.

With "Unofficial" or "Unknown" I'd be able to only show official chapters, as I prefer to read official translations if possible. (if the website has them marked as official of course)

Without that info that filtering wouldn't be possible anymore, unless someone makes the scanlator filter support both white- and blacklists.